ssl/ssl_lib.c

```diff
@@ -2295,6 +2295,14 @@ int SSL_renegotiate_pending(const SSL *s)
     return (s->renegotiate != 0);
 }
 
+int SSL_new_session_ticket(SSL *s)
+{
+	if (SSL_in_init(s) || !s->server || !SSL_IS_TLS13(s))
```

Comment:
I feel like there should be another thing to check here, but this matches the documentation and I am not sure what the other check would be...

Reply:
During the early data phase we come out of init for a while - but the first handshake has not yet completed. Therefore we might want a check of `SSL_IS_FIRST_HANDSHAKE` here (which returns true if still in the first handshake regardless of the "init" value).

Reply:
That is almost certainly the other thing I was thinking of; thanks!